firebase-project/src/router/index.js

```diff
@@ -34,15 +35,12 @@ const routes = [
   },
 
   {
-    path: "/post/:uid",
+    path: "/post/:id",
     name: "single-post",
-    component: () =>
-      import(
```

Comment:
why did you remove the impro statement and directly use the component @mukeshsin ?

Reply:
done